.travis.yml

```diff
@@ -14,9 +17,14 @@ before_script:
 
     # Workaround for Selenium #3280 issue
   - "sudo sed -i 's/^127\\.0\\.0\\.1.*$/127.0.0.1 localhost/' /etc/hosts"
+
+  - "cd node_modules/pouchdb && npm install && cd -"
+  - "rm -fr node_modules/pouchdb/node_modules/express-pouchdb"
+  - "ln -s ../../express-pouchdb node_modules/pouchdb/node_modules/express-pouchdb"
+
   - "cd node_modules/pouchdb-server && npm install && cd -"
   - "rm -fr node_modules/pouchdb-server/node_modules/express-pouchdb"
-  - "ln -s ../../.. node_modules/pouchdb-server/node_modules/express-pouchdb"
+  - "ln -s ../../express-pouchdb node_modules/pouchdb-server/node_modules/express-pouchdb"
```

Comment:
I tried pretty much all variations I could up with, including copying over what it was and just replacing pouchdb-server with pouchdb (that was the first try). The current commit is just another experiment that failed.